src/listenerManager.js

```diff
@@ -112,6 +112,9 @@ module.exports = function(dataLayerManager) {
    * @private
    */
   function _callHandler(listener, item, isPastItem) {
+    // Fixes #91. Do not trigger event during initialization when event was pushed after adding listener and before DL initialization
```

Comment:
Just having the comment without the link to the issue id should be good enough. Users can browse the history in git to get more context if needed.